config/default-test.js

```diff
@@ -1,9 +1,12 @@
 'use strict';
 
+const _ = require('lodash');
 const path = require('path');
 
 const defaultCfg = require('./default');
 
+const config = _.cloneDeep(defaultCfg);
```

Comment:
I see. So in this case, I need to make a change to `config.content` so I should cloneDeep just that object you're saying?